bindata/network/ovn-kubernetes/managed/multi-zone-interconnect/ovnkube-control-plane.yaml

```diff
@@ -13,36 +13,36 @@ spec:
   minAvailable: {{.OVN_MIN_AVAILABLE}}
   selector:
     matchLabels:
-      app: ovnkube-master
+      app: ovnkube-control-plane
 ---
 {{ end }}
-kind: StatefulSet
+kind: Deployment
 apiVersion: apps/v1
 metadata:
-  name: ovnkube-master
+  name: ovnkube-control-plane
   namespace: {{.HostedClusterNamespace}}
   annotations:
     network.operator.openshift.io/cluster-name:  {{.ManagementClusterName}}
     kubernetes.io/description: |
-      This daemonset launches the ovn-kubernetes controller (master) networking components.
+      This deployment launches the ovn-kubernetes control plane components.
     release.openshift.io/version: "{{.ReleaseVersion}}"
   labels:
     # used by PodAffinity to prefer co-locating pods that belong to the same hosted cluster.
     hypershift.openshift.io/hosted-control-plane: {{.HostedClusterNamespace}}
 spec:
+  replicas: 3
```

Comment:
@ricky-rav we need to use number of replicas as `1` when deploying in non-ha hypershift.

Reply:
heh, take a look at the dup line 39 below! :)